TaskManagementSystem/src/main/java/com/hillel/task_management_system/controller/TaskController.java

```diff
@@ -0,0 +1,72 @@
+package com.hillel.task_management_system.controller;
+
+import com.hillel.task_management_system.enums.Priority;
+import com.hillel.task_management_system.enums.Status;
+import com.hillel.task_management_system.model.Task;
+import com.hillel.task_management_system.service.TaskService;
+import com.hillel.task_management_system.service.UserService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class TaskController {
+
+    private final TaskService taskService;
+
+    private final UserService userService;
```

Comment:
Why did you autowire UserSrvice into TaskController?

Reply:
Sorry, I didn't get what you mean

Reply:
I've used userService in a couple of methods of taskService: showUserTasks and deleteUserFromListOfTask.